sddl/sddlHelper_linux.go

```diff
@@ -870,25 +871,6 @@ func aceRightsToString(aceRights uint32) string {
 	return fmt.Sprintf("0x%x", aceRights)
 }
 
-// Does the aceType correspond to an object ACE?
-// We don't support object ACEs.
-func isObjectAce(aceType byte) bool {
-	switch aceType {
```

Comment:
This code is very new and has not seen much test&dev since it was created.
Based on how the code is used/tested, we might have the need to use this function.
That function being around let's developers know that this can be used to check, which IMO is pretty useful for new code.